frontend/components/buttons/Button/_styles.scss

```diff
@@ -128,18 +127,17 @@ $base-class: 'button';
 
   &--gradient {
     background-color: transparent;
-    background-image: linear-gradient(134deg, $gradient-start 0%, $gradient-end 100%);
+    background-image: linear-gradient(134deg, $gradient-start 0%, $gradient-end 0%);
     color: $white;
     font-size: $large;
     font-weight: $light;
     letter-spacing: 4px;
     padding: $medium;
     width: 100%;
     height: auto;
-    box-shadow: inset 0 -3px 0 0 rgba(0, 0, 0, 0.25), 0 -2px 3px 0 rgba(0, 0, 0, 0.15);
 
     &:active {
-      box-shadow: 0 1px 0 #734893, 0 -2px 0 #d1d9e9;
+      box-shadow: none;
```

Comment:
If you're removing the box-shadow from `--gradient` than setting it to "none" is not needed. You can remove this whole :active section.